CHANGES.rst

```diff
@@ -94,6 +94,8 @@ cube_build
 
 - Remove certain WCS keywords that are irrelevant after ``cube_build``. [#6032]
 
+- Fix some typos in the the arguments documentation. [#6077]
```

Comment:
this needs to move up to the top section of the log, since it'll be included in 1.2.1

Reply:
Whoops, missed that section.  Fixed.